sympy/solvers/ode/tests/test_systems.py

```diff
@@ -781,11 +951,166 @@ def test_linear_3eq_order1_type4_slow():
+    2*b1*b3**3*m + 2*I*b2**3*m*sqrt(b1**2 + b2**2 + b3**2) + 2*I*b2*b3**2*m*sqrt(b1**2 + b2**2 + b3**2))
+    - b2**2*b3**2*e1*q*exp(q*t*sqrt(-b1**2 - b2**2 - b3**2)/m)/(2*b1**3*b3*m + 2*I*b1**2*b2*m*sqrt(b1**2
+    + b2**2 + b3**2) + 2*b1*b2**2*b3*m + 2*b1*b3**3*m + 2*I*b2**3*m*sqrt(b1**2 + b2**2 + b3**2) +
+    2*I*b2*b3**2*m*sqrt(b1**2 + b2**2 + b3**2)) - I*b2**2*b3*e2*q*sqrt(b1**2 + b2**2 +
+    b3**2)*exp(q*t*sqrt(-b1**2 - b2**2 - b3**2)/m)/(2*b1**3*b3*m + 2*I*b1**2*b2*m*sqrt(b1**2 + b2**2 +
+    b3**2) + 2*b1*b2**2*b3*m + 2*b1*b3**3*m + 2*I*b2**3*m*sqrt(b1**2 + b2**2 + b3**2) +
+    2*I*b2*b3**2*m*sqrt(b1**2 + b2**2 + b3**2)) + b2**2*e3*q*exp(q*t*sqrt(-b1**2 - b2**2 -
+    b3**2)/m)/(2*b1**2*m + 2*b2**2*m + 2*b3**2*m), t)
+    x_2 = 1/(2*b1**2*m*exp(q*t*sqrt(-b1**2 - b2**2 - b3**2)/m) + 2*b2**2*m*exp(q*t*sqrt(-b1**2 - b2**2 - b3**2)/m) +
+            2*b3**2*m*exp(q*t*sqrt(-b1**2 - b2**2 - b3**2)/m))
+    x_3 = exp(q*t*sqrt(-b1**2 - b2**2 - b3**2)/m)
+    x_4 = sqrt(b1**2 + b2**2 + b3**2)
+    x_5 = Integral(b1**2*e3*q*x_2 - b1*b3*e1*q*x_2 + I*b1*e2*q*x_2*x_4 + b2**2*e3*q*x_2 - b2*b3*e2*q*x_2 - I*b2*e1*q*x_2*x_4, t)
+    x_6 = Integral(b1*b3*e1*q/(b1**2*m + b2**2*m + b3**2*m) + b2*b3*e2*q/(b1**2*m + b2**2*m + b3**2*m) + b3**2*e3*q/
+                (b1**2*m + b2**2*m + b3**2*m), t)
+    x_7 = 1/(b1**2*x_3*x_4 + b2**2*x_3*x_4)
+    x_8 = 1/(b1**2*x_4 + b2**2*x_4)
+    x_9 = 1/(b1**2*x_3 + b2**2*x_3)
+    x_10 = sqrt(-b1**2 - b2**2 - b3**2)
+    x_11 = 1/(b1**2 + b2**2)
+    x_12 = b3**2
+    x_13 = 1/b3
+    x_14 = b2**3
+    x_15 = b1**2
+    x_16 = C1*b1*x_13 - C2*b1*b3*x_4*x_7 - I*C2*b2*x_12*x_7 - I*C2*b2*x_15*x_7 - I*C2*x_14*x_7 - C3*b1*b3*x_3*x_4*x_8 + \
+           I*C3*b2*x_12*x_3*x_8 + I*C3*b2*x_15*x_3*x_8 + I*C3*x_14*x_3*x_8 - b1*b3*x_1*x_4*x_7 - b1*b3*x_3*x_4*x_5*x_8 \
+           + b1*x_13*x_6 - I*b2*x_1*x_12*x_7 - I*b2*x_1*x_15*x_7 + I*b2*x_12*x_3*x_5*x_8 + I*b2*x_15*x_3*x_5*x_8 - \
+           I*x_1*x_14*x_7 + I*x_14*x_3*x_5*x_8
+    x_17 = C1*b2*x_13 + C2*b1*x_10*x_9 - C2*b2*b3*x_9 - I*C3*b1*x_11*x_3*x_4 - C3*b2*b3*x_11*x_3 + b1*x_1*x_10*x_9 - \
+           I*b1*x_11*x_3*x_4*x_5 - b2*b3*x_1*x_9 - b2*b3*x_11*x_3*x_5 + b2*x_13*x_6
+    sol = [Eq(v1(t), x_16), Eq(v2(t), x_17), Eq(v3(t), C1 + C3*x_3 + x_3*x_5 + x_6 + (C2 + x_1)*exp(-q*t*x_10/m))]
+    assert dsolve_sol == sol
+
+
 @slow
-def test_linear_3eq_order1_type4_skip():
+def test_linear_new_order1_type2_de_lorentz_fail():
     if ON_TRAVIS:
         skip("Too slow for travis.")
 
+    eqs, sol, _ = _de_lorentz_solution()
+
+    assert checksysodesol(eqs, sol) == (True, [0, 0, 0])
+
+
+def _neq_order1_type2_slow():
+    RC, t, C, Vs, L, R1, V0, I0 = symbols("RC t C Vs L R1 V0 I0")
+    V = Function("V")
+    I = Function("I")
+    system = [Eq(V(t).diff(t), -1 / RC * V(t) + I(t) / C), Eq(I(t).diff(t), -R1 / L * I(t) - 1 / L * V(t) + Vs / L)]
+
+    return system, dsolve(system), [RC, t, C, Vs, L, R1, V0, I0, V, I]
```

Comment:
It would be better to define the expected solution here and return `system, funcs, sol` and let the caller call `dsolve`. The way this is set up `dsolve` gets called twice because of the two tests that call this function.